bindings/azure/storagequeues/storagequeues_test.go

```diff
@@ -0,0 +1,175 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package storagequeues
+
+import (
+	"context"
+	"os"
+	"syscall"
+	"testing"
+	"time"
+
+	"github.com/dapr/components-contrib/bindings"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+type MockHelper struct {
+	mock.Mock
+}
+
+func (m *MockHelper) Init(accountName string, accountKey string, queueName string) error {
+	retvals := m.Called(accountName, accountKey, queueName)
+	return retvals.Error(0)
+}
+
+func (m *MockHelper) Write(data []byte) error {
+	return nil
+}
+
+func (m *MockHelper) Read(ctx context.Context, consumer *consumer) error {
+	return nil
+}
+
+func TestWriteQueue(t *testing.T) {
+
+	mm := new(MockHelper)
+	mm.On("Init", mock.AnythingOfType("string"), mock.AnythingOfType("string"), mock.AnythingOfType("string")).Return(nil)
+
+	a := AzureStorageQueues{helper: mm}
+
+	m := bindings.Metadata{}
+	m.Properties = map[string]string{"accountKey": "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==", "queueName": "queue1", "accountName": "devstoreaccount1"}
```

Comment:
There's a secret here..

Reply:
That looks like the storage emulator.

Reply:
Correct its just the storage emulator, harmless